### ISSUE TITLE
Testing Catheter Response in Pressure Chamber

### ISSUE THREAD

**Issue by yashasaxena** [Duke-BME-Design/Ureter#28]
should check what goes on w voltage readings with
1) moving catheter down a column of water
2) persitalsis stimulation

**Comment by yashasaxena** [comment]
So, I detached the U-tube manometer from the respirator and attached a syringe instead. The respirator cc/stroke was to string for the little U-dude. I verified that the catheter was working prior to running this test.

I held the catheter constant at the 5 mm Hg point in the U-tube

So, in a 50 second run I tried to pulse the syringe about 1x per second:
Trial 1:  
![image](https://user-images.githubusercontent.com/8584289/37437086-a142577e-27c1-11e8-9e19-4ea6478a6ae0.png)
Trial 2: 
![image](https://user-images.githubusercontent.com/8584289/37437281-b2069e3e-27c2-11e8-9c28-3f8f3d98a9b3.png)

Looks like our low pass filters are doing some work here. There might have been some freak syringe accidents at 1.0 and 1.6 seconds. 

I think the major changes in voltage are sadly related to drift, but peristalsis at this level doesn't look like it is a problem. Compare to inserting from 0-5mm Hg without peristalsis, looks like similar "noisiness" around drift: 
![image](https://user-images.githubusercontent.com/8584289/37437405-6c5d98e6-27c3-11e8-8fbe-d930dbbfe33d.png)
